packages/x-components/src/views/home/Home.vue

```diff
@@ -340,6 +340,32 @@
               </ResultsList>
             </LocationProvider>
 
+            <SemanticQueries #default="{ queries }">
```

Comment:
Shouldn't we wrap them inside a LocationProvider and set there origin (no-results)?